### DISCUSSION TITLE
Documentation updates for Stargate v2

### DISCUSSION THREAD

**Discussion by jeffreyscarpenter** [stargate/stargate#1515] [General]
This discussion is to help capture documentation changes we might need for Stargate v2 to make sure we don't miss things.

Examples: 
- instructions for running SGv2 in Docker compose and Kubernetes
- guidance on how to size a cluster - how many instances of each API and coordinator do you need?

**Comment by dougwettlaufer** [comment]
Some other suggestions:

- How to run a local "dev mode"
- Net new docs around helm
- Enable/disable specific APIs
